src/components/ProfileDropdown.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useContext } from 'react';
+import { LogoutIcon } from '@heroicons/react/solid';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { UserContext } from '../hook/useAuth';
+
+function ProfileDropdown({
+  handleShowProfileDropdown,
+}: {
+  handleShowProfileDropdown: any;
+}) {
+  const { logout } = useContext(UserContext);
+  const { t } = useTranslation();
+  return (
+    <div className="w-screen h-screen fixed top-0 left-0 z-50 px-4">
+      <div
+        className="bg-dark-20 w-full h-full absolute top-0 left-0 z-1"
+        role="button"
+        aria-label="background"
+        tabIndex={0}
+        onClick={handleShowProfileDropdown}
+        onKeyDown={handleShowProfileDropdown}
+        data-testid="keyClick"
+      />
+      <div className="absolute top-[60px] right-0 left-0 ml-auto px-2 md:mx-0 md:left-auto md:right-[6px] z-2  w-full max-w-[220px] h-[calc(100%-70px)]">
+        <div className="flex flex-col flex-nowrap w-full h-max max-h-full bg-white shadow-xl border  dark:border-0 dark:bg-dark-tertiary rounded-[20px]">
+          <div className="flex flex-col w-full p-3 border-border-dark dark:border-white border-b-[0.5px]">
+            <p className="font-bold dark:text-white">John Doe</p>
+            <p className="font-bold text-sm text-gray-700 dark:text-gray-300">
+              Johndoe@gmail.com
+            </p>
+          </div>
+          <div
+            className="flex flex-col w-full overflow-auto"
+            data-testid="notificationsContainer"
+          >
+            <div className="w-full p-3 pl-4 border-border-dark dark:border-white border-b-[0.5px]">
+              <div className="flex flex-row justify-between align-center gap-x-[20px] ">
+                <div className="flex flex-col w-full gap-[5px] cursor-pointer">
+                  <p className="font-semibold text-gray-600 dark:text-white p-1 hover:bg-gray-600 hover:text-gray-200 dark:hover:bg-gray-300 dark:hover:text-gray-900">
```

Comment:
Give the `menu` the full width to fix this issue of hover effects different for the `signout` to the others

![image](https://user-images.githubusercontent.com/28519395/185965514-aa70ec45-0599-49d1-900a-5bf3d9dbcfdf.png)